server/utils/tests/openAITest.js

```diff
@@ -0,0 +1,24 @@
+import assert from "assert";
+import runPrompt from "../openAI.js";
+
+describe("open ai api tests", () => {
+  describe("translation test", () => {
+    it("should translate hello to שלום", async function () {
+      const result = await runPrompt(
```

Comment:
what are you testing here? I know the openAi api works. you need to use function/endpoint that we created